server/server.go

```diff
@@ -48,10 +51,22 @@ func New(c *Config) (*Server, error) {
 		return nil, errors.Wrap(err, "failed to initialize base server")
 	}
 
+	// Default to 50 MB for the cache size
+	maxSize := int64(50 * 1024 * 1024)
```

Comment:
Is there a nice stdlib package that we can use so operators could write something like `50MB` in the config file?

Reply:
Maybe https://godoc.org/github.com/inhies/go-bytesize?

Reply:
Yeah, nothing in the standard library, but I'd be fine adding either the library you mentioned or one of these:

- https://godoc.org/github.com/docker/go-units
- https://godoc.org/github.com/c2h5oh/datasize

Because `c2h50h/datasize` implements `UnmarshalText`, you _might_ be able to reference it directly in the configuration struct and have the YAML library automatically parse it... not sure if `gopkg.in/yaml.v2` supports that or if it is only `encoding/json`.

Reply:
Oh nice find on the later package, which seems to work as expected.